compiler/fir/resolve/src/org/jetbrains/kotlin/fir/declarations/InlineClassesUtils.kt

```diff
@@ -11,7 +11,10 @@ import org.jetbrains.kotlin.fir.resolve.substitution.createTypeSubstitutorByType
 import org.jetbrains.kotlin.fir.resolve.toSymbol
 import org.jetbrains.kotlin.fir.resolve.transformers.ensureResolved
 import org.jetbrains.kotlin.fir.symbols.impl.FirRegularClassSymbol
-import org.jetbrains.kotlin.fir.types.*
```

Comment:
As the refactoring was reverted, you could revert this part and remove this file from the commit at all? :)

Reply:
Well, of course, specific imports are better, so feel free to leave these as-is.

Reply:
Ah it's my muscle memory reformatting the file after reverting. Let me untouch this file. to leave the history cleaner.